datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -163,14 +163,16 @@ WHERE conflated_balances.vega_time < (SELECT coalesce(min(balances.vega_time), '
 
 create table ledger
 (
-    ledger_entry_time       TIMESTAMP WITH TIME ZONE NOT NULL,
-    account_from_id bytea                      NOT NULL,
-    account_to_id   bytea                      NOT NULL,
-    quantity        HUGEINT                  NOT NULL,
-    tx_hash         BYTEA                    NOT NULL,
-    vega_time       TIMESTAMP WITH TIME ZONE NOT NULL,
-    transfer_time   TIMESTAMP WITH TIME ZONE NOT NULL,
-    type            TEXT,
+    ledger_entry_time              TIMESTAMP WITH TIME ZONE NOT NULL,
+    account_from_id                bytea                    NOT NULL,
+    account_to_id                  bytea                    NOT NULL,
+    quantity                       HUGEINT                  NOT NULL,
+    tx_hash                        BYTEA                    NOT NULL,
+    vega_time                      TIMESTAMP WITH TIME ZONE NOT NULL,
+    transfer_time                  TIMESTAMP WITH TIME ZONE NOT NULL,
+    sender_post_transfer_balance   HUGEINT                  NOT NULL,
```

Comment:
I think you may need to add this as a migration? @ettec

Reply:
Never did that until now when adding columns.